ui/src/main/webapp/admin-tools/wcpm/required-claims.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import Flexbox from 'flexbox-react'
+
+import { List, ListItem } from 'material-ui/List'
+
+import AttributeMapper from 'components/AttributeMapper'
+
+import { Subtitle, Description } from './components'
+
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn
+} from 'material-ui/Table'
+
+const Layout = ({ children }) => (
+  <div>
+    <Flexbox flexDirection='row' flexWrap='wrap' style={{ padding: '15px 5px' }}>
+      <Flexbox flex='1' flexDirection='column' style={{ minWidth: 300, marginRight: 20 }}>
+        <div>
+          <Subtitle>Required Subject Claims</Subtitle>
+          <Description>
+            List of claims required for each Web Context.
```

Comment:
`The required attributes and values a subject must have to access the set of Web Contexts.` might be a bit more descriptive.